### PR TITLE
New TC for PoliciesManager changes status to UPDATING

### PR DESCRIPTION
**Pull request by istoimenova**
To run the script you need to replace
user_modules/shared_testcases/testCasesForPolicyTable.lua
user_modules/shared_testcases/testCasesForPolicyTableSnapshot.lua
user_modules/shared_testcases/testCasesForBuildingSDLPolicyFlag.lua
from #32. Common functions are under review.

@maria-mih please review the script according to template.

### CODE REVIEW

File: test_scripts/Polices/Policy_Table_Update/020_ATF_P_TC_PoliciesManager_changes_status_UPDATING.lua [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+function Test:TestStep_Sending_PTS_to_mobile_application()
+  local is_test_fail = false
+  local correlationId = self.mobileSession:SendRPC("RegisterAppInterface", config.application1.registerAppInterfaceParams)
+
+  EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered", { application = { appName = config.application1.appName } })
+  :Do(function(_,data)
+      local hmi_app_id = data.params.application.appID
+      EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", {status = "UPDATE_NEEDED"})
+
+      testCasesForPolicyTableSnapshot:create_PTS(true,
```
Comment (istoimenova) [inline test_scripts/Polices/Policy_Table_Update/020_ATF_P_TC_PoliciesManager_changes_status_UPDATING.lua]: @istoimenova create_PTS should be renamed as verify_PTS

File: test_scripts/Polices/Policy_Table_Update/020_ATF_P_TC_PoliciesManager_changes_status_UPDATING.lua [hunk 2]
```diff
@@ -0,0 +1,167 @@
+---------------------------------------------------------------------------------------------
```
Comment (maria-mih) [inline test_scripts/Polices/Policy_Table_Update/020_ATF_P_TC_PoliciesManager_changes_status_UPDATING.lua]: Review started

File: test_scripts/Polices/Policy_Table_Update/020_ATF_P_TC_PoliciesManager_changes_status_UPDATING.lua [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+-- Requirements summary:
+-- [PolicyTableUpdate] PoliciesManager changes status to “UPDATING”
+-- [HMI API] SystemRequest request/response
+--
+-- Description:
```
Comment (maria-mih) [inline test_scripts/Polices/Policy_Table_Update/020_ATF_P_TC_PoliciesManager_changes_status_UPDATING.lua]: Please check description

File: test_scripts/Polices/Policy_Table_Update/020_ATF_P_TC_PoliciesManager_changes_status_UPDATING.lua [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+-- PTU is requested.
+-- SDL->HMI: SDL.OnStatusUpdate(UPDATE_NEEDED)
+-- SDL->HMI:SDL.PolicyUpdate(file, timeout, retry[])
+-- HMI -> SDL: SDL.GetURLs (<service>)
+-- HMI->SDL: BasicCommunication.OnSystemRequest ('url', requestType:PROPRIETARY, appID="default")
+-- SDL->app: OnSystemRequest ('url', requestType:PROPRIETARY, fileType="JSON", appID)
+-- 2. Performed steps
+--
+-- Expected result:
+-- SDL->HMI: SDL.OnStatusUpdate(UPDATING) right after SDL->app: OnSystemRequest
```
Comment (maria-mih) [inline test_scripts/Polices/Policy_Table_Update/020_ATF_P_TC_PoliciesManager_changes_status_UPDATING.lua]: Please check if  message is really sent after OnSystemRequest

**Comment by istoimenova** [comment]
@LuxoftAKutsan please merge.

**Pull request merged**